manifest/service_test.go

```diff
@@ -92,3 +92,15 @@ func TestSyncPaths(t *testing.T) {
 		}
 	}
 }
+
+func TestUseSecureEnvironment(t *testing.T) {
+	m, err := manifestFixture("secure-env")
+	if err != nil {
+		assert.FailNow(t, fmt.Sprintf("failed to read fixture: %s", err.Error()))
```

Comment:
There's a `require.NoError()` function that wraps this error check logic with nice output.